tests/load/utils.py

```diff
@@ -573,14 +573,6 @@ def destinations_configs(
 
     # add local filesystem destinations if requested
     if local_filesystem_configs:
-        destination_configs += [
-            DestinationTestConfiguration(
-                destination_type="filesystem",
-                bucket_url=FILE_BUCKET,
-                file_format="insert_values",
```

Comment:
why remove this?

Reply:
Because i had special code testing the insert_values and these are implicitly tested on many destinations